appkit/src/test/scala/org/ergoplatform/appkit/ErgoAuthSpec.scala

```diff
@@ -0,0 +1,41 @@
+package org.ergoplatform.appkit
+
+import org.scalatest.{Matchers, PropSpec}
+import org.scalatestplus.scalacheck.ScalaCheckDrivenPropertyChecks
+import scorex.util.Random
+import sigmastate.interpreter.HintsBag
+
+import java.nio.charset.StandardCharsets
+
+class ErgoAuthSpec extends PropSpec with Matchers with ScalaCheckDrivenPropertyChecks
+  with AppkitTestingCommon {
+
+  property("ErgoAuth address roundtrip") {
+    // server side
+    val serializedSigmaBoolean = ErgoAuthUtils.serializeSigmaBoolean(address)
```

Comment:
It is better to use SigmaBoolean here and also below.

Reply:
Done